cmd/api-resources_test.go

```diff
@@ -64,11 +66,34 @@ func TestListObjectsV2Resources(t *testing.T) {
 			fetchOwner:   true,
 			maxKeys:      1000,
 			encodingType: "gzip",
+			errCode:      "",
+		},
+		{
+			values: url.Values{
+				"prefix":             []string{"photos/"},
+				"continuation-token": []string{""},
+				"start-after":        []string{"start-after"},
+				"delimiter":          []string{"/"},
+				"fetch-owner":        []string{"true"},
+				"encoding-type":      []string{"gzip"},
+			},
+			prefix:       "",
+			token:        "",
+			startAfter:   "",
+			delimiter:    "",
+			fetchOwner:   false,
+			maxKeys:      0,
+			encodingType: "",
+			errCode:      "InvalidArgument",
 		},
 	}
 
 	for i, testCase := range testCases {
-		prefix, token, startAfter, delimiter, fetchOwner, maxKeys, encodingType := getListObjectsV2Args(testCase.values)
+		prefix, token, startAfter, delimiter, fetchOwner, maxKeys, encodingType, errCode := getListObjectsV2Args(testCase.values)
+		apiError := getAPIError(errCode)
```

Comment:
@Praveenrajmani,
We actually do **not** need this line and then check the string.

The only change we need is the following line where the expected `errCode` is set.
That is;
```
-   errCode:      125,
+   errCode:      ErrIncorrectContinuationToken,
``` 

I tested it this way and it workes.
Why do we need to check the error code string, `getAPIError(errCode)` ? 
Is there a reason I cannot see?

Reply:
@ebozduman Well the actual code for `ErrIncorrectContinuationToken` is `InvalidArgument` . 
So i though of checking with this.
But this looks meaningful , lets check with `ErrContinuationToken` Itself . Will do the change.

Reply:
@ebozduman  changed